src/validators/node-contacts.validator.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const joi = require('@hapi/joi');
+
+const nodeSchema = {
+  POST: joi
+    .object({
+      contacts: joi.array().items(
+        joi.object({
+          type: joi
+            .string()
+            .valid('direct-bluetooth', 'direct-network', 'indirect', 'manual')
+            .required(),
+
+          timestamp: joi
+            .date()
+            .iso()
```

Comment:
Is this for the contact date? I suggest we use `timestamp` here or another `datetime` format (e.g. `yyyy-MM-dd 23:00:00`). I think we'd want this to be as accurate as possible.

Reply:
It seems to allow ISO formatted timestamp strings like `2020-04-01T14:36:15.232+08:00`.

Reply:
Wait, do you mean to force them to include time and not just the date?

i.e. allow `2020-04-01T14:36:15.232+08:00` but not `2020-04-01`

Reply:
That's correct. Can we do that? Or maybe we should just switch it to `timestamp` and let the frontend handle the `datetime` formatting? What do you think?

Reply:
Ok. I think it makes sense.

Reply:
I think I'll do the datetime formatting since it is more descriptive at first glance (`1585759724 ` vs `2020-04-01T16:48:44+00:00`)

Reply:
Sorry for the late reply. I agree with you sir. This PR looks good already, please merge once this validation change is settled. Thanks! 🙇